jdaviz/configs/specviz/helper.py

```diff
@@ -39,27 +39,32 @@ def __init__(self, *args, **kwargs):
                                handler=self._redshift_listener)
 
     def load_spectrum(self, data, data_label=None, format=None, show_in_viewer=True,
-                      spectrum_viewer_reference_name=None):
```

Comment:
Hmm you added `spectrum_viewer_reference_name` in #1681 but it looks unused. So I think it's ok to remove.

Reply:
Correct, it should have been removed during revisions of #1681. Removing it now.